src/stacker/ConcreteStacker.ts

```diff
@@ -47,7 +48,7 @@ export class ConcreteStacker implements Stacker {
   }
 
   loadRepositoryInformation(): void {
-    // TODO: Implement
+    return this.sourceControl.loadRepositoryInformation();
```

Comment:
nit: this function expects `void`
```suggestion
    this.sourceControl.loadRepositoryInformation();
```